bazel/repositories.bzl

```diff
@@ -195,6 +197,14 @@ def _envoy_api_deps():
         name = "http_api_protos",
         actual = "@googleapis//:http_api_protos",
     )
+    sub_bind_targets = [
+        ("auth", "auth"),
+    ]
+    for t in sub_bind_targets:
+        native.bind(
+            name = "envoy_api_sub_" + t[0],
```

Comment:
What is the goal here?

Reply:
This is for config(s) under directory `auth` in [data-plane-api](https://github.com/envoyproxy/data-plane-api/tree/master/api/auth)
Need to be able to specify dependency on the proto's in that directory. for example [here](https://github.com/colabsaumoh/envoy/blob/4c3e58fd4b221ba61c3d28a824ae618b10e480ce/include/envoy/ext_authz/BUILD#L15)
I'll re-write it using the idioms already in place for filters.